trino/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergHiveMetastoreAutoCleanMetadataFile.java

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
> trino/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergHiveMetastoreAutoCleanMetadataFile.java

The file path is wrong. It should be `plugin/trino-iceberg/...`, remove a leading `trino`.